work around conflicting target names in cargo test.
only build the rust lib since we don't actually install
the c or static library at the moment. 
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -72,7 +72,6 @@
 
 [lib]
 path = "src/lib.rs"
-crate-type = ["lib", "staticlib", "cdylib"]
 
 [profile.dev]
 debug = 1
